### ISSUE TITLE
subscription already opened

### ISSUE THREAD

**Issue by mikedilger** [Yonle/bostr#16]
I update subscriptions (essentially starting over with the same subscription id) and get messages like this:

```
[RELAY REDACTED]: Closed: temp_augments: duplicate: subscription already opened
```

NIP-01 says: "...or a new REQ is sent using the same <subscription_id>, in which case relay MUST overwrite the previous subscription."  It's kind-of a shorthand for CLOSE and then REQ again.

**Comment by Yonle** [comment]
I may put a blame in their provided example:
> Some examples:
> `["CLOSED", "sub1", "duplicate: sub1 already opened"]`

**Comment by Yonle** [comment]
Solved as per be9b0698b881ba251afad49a25aed1737301ea38

*Wonder who the heck put that example. But anyway, well.*

**Comment by mikedilger** [comment]
hah!  I hadn't noticed that example.  That needs fixing.
